src/main/scala/chisel3/util/SRAM.scala

```diff
@@ -575,21 +577,20 @@ object SRAM {
     }
 
     // Add metadata to the SRAM.
-    val description = _out.description
-    description.depth := Property(size)
-    description.dataWidth := Property(tpe.getWidth)
-    description.masked := Property(isVecMem)
-    description.read := Property(numReadPorts)
-    description.write := Property(numWritePorts)
-    description.readwrite := Property(numReadwritePorts)
-    description.maskGranularity := Property(
+    _out.description.get.depth := Property(size)
+    _out.description.get.dataWidth := Property(tpe.getWidth)
+    _out.description.get.masked := Property(isVecMem)
+    _out.description.get.read := Property(numReadPorts)
+    _out.description.get.write := Property(numWritePorts)
+    _out.description.get.readwrite := Property(numReadwritePorts)
+    _out.description.get.maskGranularity := Property(
```

Comment:
```suggestion
    val description =  _out.description.get
    description.depth := Property(size)
    description.dataWidth := Property(tpe.getWidth)
    description.masked := Property(isVecMem)
    description.read := Property(numReadPorts)
    description.write := Property(numWritePorts)
    description.readwrite := Property(numReadwritePorts)
    description.maskGranularity := Property(
```
This diff can be simpler (and more elegant) if we stick with `val description`.